app/models/business_partner.rb

```diff
@@ -11,13 +11,25 @@
 #
 
 class BusinessPartner < ActiveRecord::Base
+  include ActiveModel::Serializers::JSON
+
   belongs_to :contact_card
   has_many :input_invoices
   has_many :business_partner_accounts
   has_many :output_invoices
   has_many :daily_statements
 
   validates :name, presence: true
+  enum type: [ :buyer, :supplier, :buyer_and_supplier ]
```

Comment:
Is this `type` field working? Because we had problems in XML project with fields named `type` in the database...